src/controller/batches/batch.service.ts

```diff
@@ -21,6 +21,16 @@ export class BatchesService {
           sql`${zuvyBatchEnrollments.bootcampId} = ${batch.bootcampId} AND ${zuvyBatchEnrollments.batchId} IS NULL`,
         )
         .limit(batch.capEnrollment);
+      const instructorRoles = await db.select({role:sansaarUserRoles.role}).from(sansaarUserRoles).where(eq(sansaarUserRoles.userId,batch.instructorId)) 
+      const hasInstructorRole = instructorRoles.some(role => role.role === 'instructor');
```

Comment:
Avoid hardcoding values like the instructor role. Create a "helpers" folder in the constants file to store all constant values. Storing in one place allows for easier changes reflected everywhere.